db/mysql/schema.sql

```diff
@@ -306,6 +355,12 @@ CREATE TABLE `usr_user_role`
   PRIMARY KEY (`user_id`, `role_id`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 
+CREATE TABLE `usr_extuser` (
```

Comment:
yes the table for external user ID might be omitted, but we in the edem are
still waiting for exact explanation of IDP attributes, so we do not know
now if the external user id will be equal to internal one
lets go with the table, we can remove it quickly later

On Sun, Mar 22, 2015 at 12:25 PM, Tomas Knap <notifications@github.com>
wrote:

> In db/mysql/schema.sql
> <https://github.com/UnifiedViews/Core/pull/282#discussion_r26901947>:
>
> > @@ -306,6 +355,12 @@ CREATE TABLE `usr_user_role`
> >    PRIMARY KEY (`user_id`, `role_id`)
> >  ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
> >
> > +CREATE TABLE `usr_extuser` (
>
> Well, but then there is no point in having external user table, right?
> Because internal user ID is equal to CAS user id (it is provided by CAS if
> SSO is enabled). So external user table is not needed and may be removed,
> am I right?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/UnifiedViews/Core/pull/282/files#r26901947>.
>